src/draive/conversation/realtime/default.py

```
@@ -48,24 +48,22 @@ async def realtime_conversation(  # noqa: C901, PLR0915
 
     async def process_input() -> None:
         try:
+            ctx.log_debug("Starting input processing...")
             # TODO: include guardrails check
```

Pylint (3.3.7)

[warning] TODO: include guardrails check

(W0511)